src/utils/objectToFormData.ts

```diff
@@ -0,0 +1,55 @@
+import fs from "fs"
+import FormData from "form-data"
+
+export type ObjectEntry = [string, any]
+export type ObjectEntries = Array<ObjectEntry>
+
+export function objectToFormData(object: object): FormData {
+  const form = new FormData()
+
+  for (const [key, value] of Object.entries(object)) {
+    if (value == null) continue
```

Comment:
Question: shouldn't this check for `undefined` values too?

Reply:
`value == null`, with double equals (instead of three) will check for both `null` and `undefined`!